src/app/content/components/NudgeStudyTools/index.tsx

```diff
@@ -119,19 +120,40 @@ const NudgeStudyTools = ({
 
 function useTabNavigationInterceptor() {
   const ref = React.useRef<HTMLElement>();
+  const isMobile = useMatchMobileMediumQuery();
 
   React.useEffect(
     () => {
-      const abortTabbing = (event: Event) => {
+      if (isMobile) {
```

Comment:
Does this need to exclude mobile? I'm thinking external keyboard, iPad narrow window view, etc.

Reply:
Looks like keyboard navigation does need to be supported for mobile.
https://bootcamp.uxdesign.cc/keyboard-accessibility-mobile-devices-688dd73bdb7b

Reply:
The bad news is that I don't get keydown events from the browser when in responsive mode. :(

Reply:
I wonder if the xcode ios simulator would work for testing this 🤔

Reply:
It got merged, so maybe we'll want a new issue for the mobile.